packages/gcm/lib/apk/config.yaml

```diff
@@ -0,0 +1,11 @@
+registration-class: org.domokit.gcm.RegistrationIntentService$MojoService
+jars:
+  - jars/android_support_v13_java.dex.jar
+  - jars/android_support_v7_appcompat_java.dex.jar
+  - jars/android_support_v7_mediarouter_java.dex.jar
+  - jars/gcm_lib.dex.jar
+  - jars/google_play_services_default_java.dex.jar
+  - jars/interfaces_java.dex.jar
```

Comment:
It's probably better to put these jars into google cloud store or something, rather than bundling into the pub package. Unless you have a better suggestion.